docs/metadata.md

```diff
@@ -0,0 +1,14 @@
+# Volumetric metadata about the VDS
+
+Retrieve basic information related to the VDS file such as axis definitions, overall shape and size, coordinate system, import time stamp and source filename. 
+
+## Response
+On success (200) the response containing the VDS metadata.
+
+### Metadata part
```

Comment:
There is no part here, it is not a multipart response.
I think kill this at all. It just repeats itself.